table/tables/index.go

```diff
@@ -129,13 +129,14 @@ func (c *index) getIndexKeyBuf(buf []byte, defaultCap int) []byte {
 	return make([]byte, 0, defaultCap)
 }
 
-// truncateIndexValuesIfNeeded truncate the index values that be created that use only the leading part of column values.
-func (c *index) truncateIndexValuesIfNeeded(indexedValues []types.Datum) []types.Datum {
+// TruncateIndexValuesIfNeeded truncate the index values that be created that use only the leading part of column values.
```

Comment:
s/truncate/truncates

Reply:
@lilin90 We need your help to improve this comment. 😂